libraries/AP_BattMonitor/AP_BattMonitor.h

```diff
@@ -93,6 +102,10 @@ class AP_BattMonitor
     float voltage(uint8_t instance) const;
     float voltage() const { return voltage(AP_BATT_PRIMARY_INSTANCE); }
 
+    /// get voltage with sag removed (based on battery current draw and resistance)
+    float voltage_resting_estimate(uint8_t instance) const;
```

Comment:
re consting-ing the instance... but it's pass-by-value because it's a uint8.. so no need to const the argument right?